File: lesson6/c_c++/homework/homework6_5.cpp
/*
 * lesson6/c_c++/homework/homwork6_5.cpp
 */
#include <iostream>

int expand_digits_number(int numb_n) {
    int res;
    while (numb_n > 0) {
        res = numb_n % 10;
        std::cout << res;
        numb_n /= 10;
    }
    return res;
}

int main(int argc, char* argv[]) {
    int numb;
    std::cin >> numb;
    int result = expand_digits_number(numb);
    std::cout << '\n';
    return 0;
}

File: lesson6/python3/homework/homework6_5.py
#!/usr/bin/python3
#: lesson6/python3/homework/homework6_5.py
# Напишите функцию, которая (разворачивает) десятичную запись числа наоборот,
# например из 123 получается 321 или 43210 - 01234.


def expand_digits_number(numb_n):
    while numb_n > 0:
        res = numb_n % 10
        print(res, end='')
        numb_n //= 10
    # return res


numb = 543210 
result = expand_digits_number(numb)
result
print()

